gprofiler/metadata/system_metadata.py

```diff
@@ -228,7 +231,8 @@ def get_windows_network_details() -> Tuple[str, str]:
 
 def get_static_system_info() -> SystemInfo:
     if is_windows():
-        hostname = platform.node()
+        global hostname
```

Comment:
If you add `global` here - it affects the entire function which is not intended for the second branch of Linux.

let's move the Windows code to a function `_initialize_system_info_windows`, defined under `if is_windows()`?